core/src/main/java/org/apache/calcite/sql/validate/SqlConformanceEnum.java

```diff
@@ -397,6 +397,15 @@ public enum SqlConformanceEnum implements SqlConformance {
     }
   }
 
+  @Override public boolean isSortByLiteralAllowed() {
+    switch (this) {
+    case BIG_QUERY:
+      return false;
```

Comment:
is it easy to check if other dialects allow this?

Reply:
I'm not aware of a way to test this one a wide array of dialects, but if someone has a good resource for that I'd be happy to